### PR TITLE
[main] Add KEDA and TLS tests

### PR DESCRIPTION
**Pull request by pierDipi**

### CODE REVIEW

File: openshift/knative-eventing-encryption-auth.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+spec:
+  config:
+    config-features:
+      transport-encryption: "strict"
+      authentication.oidc: "enabled"
```
Comment (creydr) [inline openshift/knative-eventing-encryption-auth.yaml:5]:
```suggestion
      authentication-oidc: "enabled"
```
Can we update this here already (https://github.com/openshift-knative/eventing/pull/536)

File: openshift/e2e-common.sh [hunk 2]
```diff
@@ -100,7 +114,6 @@ function run_e2e_tests() {
 
   echo "Running e2e tests, directory ./test/e2e_channel/"
   go_test_e2e -timeout=100m -short ./test/e2e_channel/ \
-    -channels=messaging.knative.dev/v1beta1:KafkaChannel \
```
Comment (creydr) [inline openshift/e2e-common.sh:103]: Why is this not needed anymore?

Comment (pierDipi) [reply openshift/e2e-common.sh:103]: This means that this line is missing on v1.11 as I copied it from there

Comment (pierDipi) [reply openshift/e2e-common.sh:103]: I will revert

**Comment by pierDipi** [comment]
ping @matzew @creydr

**Review by creydr — APPROVED** [review]
/lgtm

**Pull request merged**